gr-uhd/examples/grc/CMakeLists.txt

```diff
@@ -13,4 +13,9 @@ install(
           uhd_siggen_gui.grc
           uhd_two_tone_loopback.grc
           uhd_wbfm_receive.grc
+          rfnoc_keep1.grc
```

Comment:
I'm not seeing that in the changed file list?

Reply:
Slipped in. Was forgotten earlier. Moved to own commit.